enterprise/app/trends/trends.tsx

```diff
@@ -186,6 +191,7 @@ export default class TrendsComponent extends React.Component<Props> {
                 secondaryName="average build time seconds"
                 secondaryLine={true}
                 separateAxis={true}
+                barClicked={this.onBarClicked.bind(this, "")}
```

Comment:
nit: usually callback function parameters are prefixed with `on` -- can you rename to `onBarClicked`?